lib/bunq/signature.rb

```diff
@@ -30,9 +30,7 @@ def verify!(response)
       sorted_bunq_headers = response.raw_headers.select(&method(:verifiable_header?)).sort.to_h.map { |k, v| "#{k.to_s.split('-').map(&:capitalize).join('-')}: #{v.first}" }
       data = %Q{#{response.code}\n#{sorted_bunq_headers.join("\n")}\n\n#{response.body}}
 
-      if response.code == 409 || response.code == 429
-        fail TooManyRequestsResponse.new(code: response.code, headers: response.raw_headers, body: response.body) 
-      end
+      return if response.code == 409 || response.code == 429
```

Comment:
I prefer early exists to be on the first line of the method. 

Also I think extracting into a separate (private) method something like `return if skip_signature_check(response.code)`

And could you please also add `Bunq::configuration.sandbox` check here?

Reply:
✅